packages/nhsuk-frontend/src/nhsuk/components/details/_index.scss

```diff
@@ -185,8 +179,7 @@ $expander-border-hover-color: $color_nhsuk-grey-3;
     }
 
     .nhsuk-details__summary::before {
-      // stylelint-disable-next-line declaration-no-important
-      display: none !important; // [1]
+      display: none;
```

Comment:
Given `// [1]` previously mentioned the JavaScript polyfill, am assuming `!important` can now be removed.